docs/usage.md

```diff
@@ -403,9 +401,94 @@ pvc/pvc1     Bound     pv100     20Gi       RWO                          11m
 
 NAME                          READY     STATUS    RESTARTS   AGE       IP             NODE
 po/pod1                       1/1       Running   0          11m       10.128.1.53    cld6b16
+```
+
+**Static provisioning** is a feature that is native to Kubernetes and that allows cluster admins to make existing storage devices available to a cluster. As a cluster admin, you must know the details of the storage device, its supported configurations, and mount options.
+
+To make existing storage available to a cluster user, you must manually create the storage device, a PV, and a PVC.
+
+Below is an example yaml specification to create Persistent Volumes using the HPE 3PAR FlexVolume driver. 
+
+```
+Note: If you have OpenShift installed, kubectl create and oc create commands can be used interchangeably when creating PVs, PVCs, and PODs.
+```
+
+Persistent volume Example
+The following creates a Persistent volume "pv-first" that provisions a compressed volume with the help of HPE 3PAR Docker Volume Plugin.
```

Comment:
Here you are claiming compressed volume creation but yaml doesn't reflect it.

Reply:
Updated the correct statement.